.github/workflows/linux_cuda_wheel.yaml

```diff
@@ -0,0 +1,149 @@
+      - name: Install torchcodec from the wheel
+        run: |
+          wheel_path=`find pytorch/torchcodec/dist -type f -name "*.whl"`
+          echo Installing $wheel_path
+          ${CONDA_RUN} python -m pip install $wheel_path -vvv
+
+      - name: Check out repo
+        uses: actions/checkout@v3
+      - name: Install ffmpeg, post build
+        run: |
+          # Ideally we would have checked for that before installing the wheel,
+          # but we need to checkout the repo to access this file, and we don't
+          # want to checkout the repo before installing the wheel to avoid any
+          # side-effect. It's OK.
+          source packaging/helpers.sh
+          assert_ffmpeg_not_installed
+
+          ${CONDA_RUN} conda install --yes conda
+          ${CONDA_RUN} conda update --all --yes
+          ${CONDA_RUN} conda config --add channels conda-forge
+          ${CONDA_RUN} conda config --set channel_priority strict
+          ${CONDA_RUN} conda config --show
+          ${CONDA_RUN} $CONDA_PREFIX/bin/conda install --yes ffmpeg=${{ matrix.ffmpeg-version-for-tests }} -c conda-forge
+          ${CONDA_INSTALL} --yes ffmpeg=${{ matrix.ffmpeg-version-for-tests }} -c conda-forge
+          ${CONDA_RUN} conda install --yes ffmpeg=${{ matrix.ffmpeg-version-for-tests }} -c conda-forge
+
+      - name: Install cuda runtime dependencies
+        run: |
+          ${CONDA_RUN} conda install --yes nvidia::libnpp nvidia::cuda-nvrtc=12.4 nvidia::cuda-toolkit=12.4 nvidia::cuda-cudart=12.4 nvidia::cuda-driver-dev=12.4
+      - name: Install test dependencies
+        run: |
+          ${CONDA_RUN} python -m pip install --pre torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
+          # Ideally we would find a way to get those dependencies from pyproject.toml
+          ${CONDA_RUN} python -m pip install numpy pytest pillow
+
+      - name: Delete the src/ folder just for fun
+        run: |
+          # The only reason we checked-out the repo is to get access to the
+          # tests. We don't care about the rest. Out of precaution, we delete
+          # the src/ folder to be extra sure that we're running the code from
+          # the installed wheel rather than from the source.
+          # This is just to be extra cautious and very overkill because a)
+          # there's no way the `torchcodec` package from src/ can be found from
+          # the PythonPath: the main point of `src/` is precisely to protect
+          # against that and b) if we ever were to execute code from
+          # `src/torchcodec`, it would fail loudly because the built .so files
+          # aren't present there.
+          rm -r src/
+          ls
+      - name: Smoke test
+        run: |
+          find $CONDA_PREFIX -name libtorchcodec\*.so | xargs ldd
+          find $CONDA_PREFIX -name libcuda\*.so\*
+          echo $CONDA_PREFIX
+          python test/decoders/manual_smoke_test.py
```

Comment:
This line and the pytest line also need to be prefixed with `${CONDA_RUN}`.

Reply:
Good catch. I didn't get to this point but fixed it now.